packages/utils/src/db.ts

```diff
@@ -89,17 +95,31 @@ export class HoprDB {
 
     let setEnvironment = false
 
-    log('using db at ', dbPath)
+    log(`using db at ${dbPath}`)
     if (forceCreate) {
       log('force create - wipe old database and create a new')
-      rmSync(dbPath, { recursive: true, force: true })
-      mkdirSync(dbPath, { recursive: true })
+      await rm(dbPath, { recursive: true, force: true })
```

Comment:
How is this to be preferred over the sync alternative?

Reply:
The sync alternative sometimes has issues with non-existent databases or considers databases non-existent even if they are existing. The Promise-based API is more preferable because it cuts the synchronous operation into multiple small chunks and because it is better suited to handle I/O coming from the OS